packages/babel-preset-babili/__tests__/preset-tests.js

```diff
@@ -0,0 +1,42 @@
+jest.autoMockOff();
+
+const babel = require("babel-core");
+const unpad = require("../../../utils/unpad");
+
+function transform(code, options = {}, sourceType = "script") {
```

Comment:
Yeah.. Like discussed here https://github.com/babel/babili/issues/76 ... separate PR for all these things ...